src/dynimage.rs

```diff
@@ -882,7 +882,11 @@ impl DynamicImage {
         dynamic_map!(*self, ref p => imageops::rotate270(p))
     }
 
-    /// Encode this image and write it to ```w```
+    /// Encode this image and write it to ```w```.
+
```

Comment:
nit: should be a `///` on this line.

Reply:
This already is a triple slash, what do you mean?

Reply:
There's no triple slash on line 886

Reply:
Oh right